### DISCUSSION TITLE
Copyright/trademark issues for a PyCharm plugin using the same name

### DISCUSSION THREAD

**Discussion by InSyncWithFoo** [microsoft/pyright#7333] [General]
I just wrote [a Pyright integration plugin for PyCharm](https://github.com/InSyncWithFoo/pyright-plugin). Its logo is derived from [this image](https://github.com/microsoft/pyright/blob/3232c2558959440a99826b9bd791b682e6a540b0/docs/img/PyrightLarge.png); I mentioned that in the README file and put a copy of the license in the root directory. Are there any other copyright/trademark issues I need to consider?

**Comment by erictraut** [comment]
I'm not aware of any other copyright or license issues. Thanks for asking, and congrats on publishing the plugin.
